frontend/common/APICall.js

```diff
@@ -41,10 +50,18 @@ class APICall{
         };
 
         return new Promise((resolve, reject) => {
-            fetch(Links.serverBaseURL + `/${path}`, options)
+            fetch(`${Links.serverBaseURL}/${path}`, options)
             .then(response => {
                 if (!response.ok) {
-                    throw new Error('Network response was not ok');
+                    if(response.status === 401){
+                        localStorage.setItem('signedIn', "false");
+                        window.location.replace('index.html');
+                    } else if(response.status === 404){
+                        throw new Error('Not found!!');//should redirect to 404 page
```

Comment:
make it redirect to 404.html even if the page doesn't exist

Reply:
Resolved